raft/tracker/progress.go

```diff
@@ -136,13 +136,13 @@ func (pr *Progress) BecomeSnapshot(snapshoti uint64) {
 
 // UpdateOnEntriesSend updates the progress on the given number of consecutive
 // entries being sent in a MsgApp, appended at and after the given log index.
-func (pr *Progress) UpdateOnEntriesSend(entries int, nextIndex uint64) error {
+func (pr *Progress) UpdateOnEntriesSend(entries int, bytes, nextIndex uint64) error {
```

Comment:
In case this isn't done by the end of the commit chain, update the comment for the `bytes` parameter.

Reply:
Done.